nucypher/policy/conditions/context.py

```diff
@@ -104,6 +104,10 @@ def is_context_variable(variable) -> bool:
     return isinstance(variable, str) and CONTEXT_REGEX.fullmatch(variable)
 
 
+def is_bytes_context_variable(variable) -> bool:
```

Comment:
Thought about it some more, and given that you would accept signature as a hex string in the context var, then it makes sense to also accept message as hex as well instead of doing the `:bytes:message:` context variable. Not really much of a difference and feels like you can always to `bytes.fromHex()` for the `message` like you do with the signature anyway.

Happy to chat more about it tomorrow in case I'm misunderstanding.